barcode_action/wizard/barcode_action.py

```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+
+class BarcodeAction(models.TransientModel):
+    _name = "barcode.action"
+    _inherit = "barcodes.barcode_events_mixin"
```

Comment:
quick one: `2020-03-09 15:01:47,324 158 WARNING openerp_test odoo.models: The model barcode.action has no _description`